example/standalone_git/main.tf

```diff
@@ -0,0 +1,35 @@
+module "nifi_registry" {
+  source = "../.."
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+  nomad_host_volume = "persistence"
```

Comment:
We do not use this variable, maybe you can remove it?

Reply:
Agree. I have removed them.